### ISSUE TITLE
[Feature Request] Only auto enter giveaways above a certain level

### ISSUE THREAD

**Issue by GregoryGHarding** [RedSparr0w/IndieGala-Helper#13]
i think it would be cool to have a setting thats like:
"do not enter giveaway with level lower than [X]"

it would help mitigate the fake giveaways with high IC values and shit away coins for nothing.

**Comment by RedSparr0w** [comment]
Thanks for the suggestion,

Thinking maybe options like

Option | Reason
:--- | :---
`max IC cost` | Avoid fake giveaways (expensive games for easy XP)
`min lvl` | Avoid most fake giveaways if higher than lvl 0, but also miss alot of giveaways
`max lvl`/`current lvl` | less load on indiegala servers if you do not attempt to enter giveaways which you are not high enough lvl for

Any other things you could think of to try avoid fakes?

**Comment by GregoryGHarding** [comment]
is there a way to read giveaway users feedback. like do not enter if they have x negative feedback or 0 feedback (fresh account not guarenteed giveaways)

**Comment by RedSparr0w** [comment]
this probably could be done. not 100% sure though because of the 'incapsula' thing indiegala has put in place to limit the number of request.

Will do some testing and see how it goes.

Would actually be pretty handy to not enter giveaways from 0 feedback received people.

**Comment by GregoryGHarding** [comment]
id go with pretty much all the options you listed. just throw a checkbox on it to enable/disable each and it works like a multi AND-IF

**Comment by RedSparr0w** [comment]
Updated status of feature request:

- [x] max IC cost

- [x] max level/current level

- [ ] min level

probably a few updates away from the last one.

**Comment by Lakston** [comment]
I think min level is more useful than max level because you can't enter GA with a lvl higher than yours but you might want to avoid GA that are lvl 0. Max IC cost would be great to avoid the 150pts+ fake GAs.

**Comment by Kureka** [comment]
Please use HTTPS for link to Steam store page, currently it's still using HTTP so sometimes redirection not working.

**Comment by RedSparr0w** [comment]
> Please use HTTPS for link to Steam store page, currently it's still using HTTP so sometimes redirection not working.

Done 👍

**Comment by Kureka** [comment]
> I think min level is more useful than max level because you can't enter GA with a lvl higher than yours but you might want to avoid GA that are lvl 0. Max IC cost would be great to avoid the 150pts+ fake GAs.

Yeah agree with that, at least please make setting so we can hide giveaways below Lev. 1
